recipes/vtk/all/conanfile.py

```diff
@@ -15,13 +15,155 @@ class VTKConan(ConanFile):
     settings = "os", "arch", "compiler", "build_type"
     exports_sources = ["CMakeLists.txt", "patches/**"]
     source_subfolder = "source_subfolder"
-    options = {"shared": [True, False], "with_qt": [True, False], "mpi": [True, False],
-                "fPIC": [True, False], "minimal": [True, False], "ioxml": [True, False],
-                "ioexport": [True, False], "mpi_minimal": [True, False]}
-    default_options = {"shared": False, "with_qt": False, "mpi": False, "fPIC": False,
-                "minimal": False, "ioxml": False, "ioexport": False, "mpi_minimal": False}
     topics = ("conan", "VTK", "3D rendering", "2D plotting", "3D interaction", "3D manipulation", 
                 "graphics", "image processing", "scientific visualization", "geometry modeling")
+    groups = ["StandAlone", "Rendering", "MPI", "Qt", "Imaging", "Tk", "Views", "Web"]
```

Comment:
If you add custom attributes, please prefix them with `_`

Reply:
Done